dotnet/src/Connectors/Connectors.Memory.Postgres/PostgresKernelBuilderExtensions.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using Microsoft.SemanticKernel.Connectors.Postgres;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Extension methods to register the Postgres <see cref="IVectorStore"/> instances on the <see cref="IKernelBuilder"/>.
+/// </summary>
+public static class PostgresKernelBuilderExtensions
```

Comment:
We actually want to retire the kernel builder extensions, and only keep the service collection extensions.
The reason is that kernel builder extensions is one of the only reasons why we need a reference to semantic kernel abstractions, and without it, we can reduce the dependency load.  So we can skip adding this file.